examples/js/rtc/WebRTCRecorder.js

```diff
@@ -0,0 +1,205 @@
+/**
+ * @author hypnosnova / https://github.com/HypnosNova
+ * @param canvas: in most time, it is renderer.domElement.
+ * @param options: {
+ *     fps: default is 60,
+ *     format: default is 'webm',
+ *     codecs: default is 'vp8'
+ * }
+ */
+
+THREE.WebRTCRecorder = function ( canvas, options ) {
+
+	if ( ! MediaSource ) {
```

Comment:
I think this will throw an error if `MediaSource` doesn't exist in the browser. You might have to use `window.MediaSource` or `typeof MediaSource` to check if it exists.

Reply:
OK, change into window.MediaSource